pkg/crc/machine/kubeconfig.go

```diff
@@ -174,12 +174,20 @@ func addContext(cfg *api.Config, ip string, clusterConfig *types.ClusterConfig,
 	if err != nil {
 		return err
 	}
-	cfg.AuthInfos[username] = &api.AuthInfo{
+
+	// append /clustername to AuthInfo
+	url, err := url.Parse(clusterConfig.ClusterAPI)
+	if err != nil {
+		return err
+	}
+	clusterUser := fmt.Sprintf("%s/%s", username, url.Host)
```

Comment:
I think it doesn't matter what we put as suffix even if we use `<username>/crc` that will also work but since we want to it should be `<username>/api-crc-testing:6443` instead `<username>/api.crc.testing:6443`

Reply:
updated to have the `.` replaced with `-`